### PR TITLE
IBP-3248 BrApiMaxPageSize

### PR DESCRIPTION
**Pull request by gelli-leafnode**
Modify for brapi max page size and default page size

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -34,4 +34,6 @@ design.runner=${design.runner}
 design.runner.license.service=${design.runner.license.service}
 
 spring.jackson.mapper.default-view-inclusion=true
-bms.version=${project.version}
\ No newline at end of file
+bms.version=${project.version}
+pagedresult.max.page.size=${pagedresult.max.page.size}
+pagedresult.default.page.size=${pagedresult.default.page.size}
```
Comment (darla-leafnode) [inline src/main/resources/application.properties:39]: Should we remove this:
`pagedresult.default.page.size=${pagedresult.default.page.size} `

Comment (gelli-leafnode) [reply src/main/resources/application.properties:39]: Will remove not used property (Defaultpagesize)

File: src/main/java/org/ibp/api/domain/common/PagedResult.java [hunk 2]
```diff
@@ -27,14 +30,14 @@
 	private String sortBy;
 	private String sortOrder;
 
-	public static final int MAX_PAGE_SIZE = 10000;
+	public static int MAX_PAGE_SIZE;
```
Comment (darla-leafnode) [inline src/main/java/org/ibp/api/domain/common/PagedResult.java:33]: I think we can convert MAX_PAGE_SIZE to a non-static local variable such as:
`protected int maxPageSize;`

Comment (gelli-leafnode) [reply src/main/java/org/ibp/api/domain/common/PagedResult.java:33]: The instances of PagedResult and BrapiPagedResult (inheriting PagedResult) are being instantiated via new, if we convert the variable MAX_PAGE_SIZE to non-static the value set during spring instantiate will be ignored. Please refer to this [link](https://mkyong.com/spring/spring-inject-a-value-into-static-variables/) for injecting value into static variable

**Pull request merged**